source/python/idetect/model.py

```diff
@@ -279,6 +279,22 @@ class Location(Base):
         'Report', secondary=report_location, back_populates='locations')
 
 
+class TermType:
+    PERSON_TERM = 'person_term'
+    PERSON_UNIT = 'person_unit'
+    STRUCTURE_TERM = 'structure_term'
+    STRUCTURE_UNIT = 'structure_unit'
+    ARTICLE_KEYWORD = 'article_keyword'
+
+
+class ReportKeyword(Base):
+    __tablename__ = 'report_keyword'
+
+    id = Column(Integer, primary_key=True)
+    description = Column(String)
+    term_type = Column(String)
```

Comment:
If this is a "term" type, should the class be called ReportTerm instead of ReportKeyword? Or should this be renamed "keyword_type"? I assume you're not using just "type" to avoid that being a reserved word in various contexts. Perhaps you could use "kind" instead?